bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1013,6 +950,131 @@ private void addMergeCommitWarningComment(List<Comment> comments) {
         pr.addComment(message);
     }
 
+    /**
+     * Add a suggestion comment to the pull requset which has an update change to direct the developers.
+     */
+    private void addUpdateChangeSuggestionComment() {
+        var existing = findComment(comments, updateChangeSuggestionMarker);
+        if (existing.isPresent()) {
+            // Only add the comment once per PR
+            return;
+        }
+        var message = new StringBuilder();
+        message.append("@");
+        message.append(pr.author().username());
+        message.append(" ");
+        var issue = Issue.fromStringRelaxed(pr.title()).flatMap(
+                value -> issueProject() != null ? issueProject().issue(value.shortId()) : Optional.empty());
+        var issueLink = "";
+        if (issue.isPresent()) {
+            issueLink = "[" + issue.get().id() + "](" + issue.get().webUrl() + ") ";
+        }
+        message.append(String.format("""
+                This is a change to the update repository or branch. Please add a comment to the main issue %s\
+                to state the related condition (the backport reason, the risk of the patch, the amount of \
+                work and so on). Below is an example for you:
+
+                ```
+                Fix Request (%s)
+                The code applies cleanly and the test in this change fails without the patch and succeeds \
+                after applying it. The risk of this backport is low because the change is little and the \
+                issue fixed by this change also exists in other update repositories.
+                ```
+
+                If you don't have permission to add a comment in JBS. Please use the command `request-approval` \
+                to provide the related content, then the bot can help you add a comment by using the content \
+                you provided. Below is an example for you:
```

Comment:
Fixed.

Reply:
I revise the message as you provided. But it seems the statement `you may use the command `/request-approval` to supply the fix request comment and automatically set the correct request label.` is not clear. Because the automatically added request label is not related to the `request-approval` command. Looking for the better statement about it.